nautobot_golden_config/routers.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from django.urls import path
+from django.apps import apps
+from django.utils.module_loading import import_string
+
+
+def _verify_import_string(class_path):
+    try:
+        import_string(class_path)
+    except ImportError:
+        return False
+    return True
+
+
+def class_factory(class_name):
+    class LocalClass(import_string(f"nautobot.core.views.generic.{class_name}")):
+        pass
+
+    return LocalClass
+
+
+def generate_view(app_label, model_name):
+    paths = []
+
+    # Setup all of the different methods that are required
+    lc_model_name = model_name.lower()
+    route = re.sub(r"(?<!^)(?=[A-Z])", "-", model_name).lower()
+
+    app_config = apps.get_app_config(app_label)
+    model = app_config.get_model(model_name)
+    model_queryset = model.objects.all()
+
+    # For now, these are hard coded, but can easily follow the same pattern
```

Comment:
I like it, I think it would be great to attach some of that to the BaseModel class to be able to automatically attach the name of these class to the model object itself and to retrieve the proper class easily as well